### ISSUE TITLE
Engaged special skill won't change in Condensed and Echoes Stats UI

### ISSUE THREAD

**Issue by Gamerhenky** [Eskuero/fehbuilder#16]
With the addition of Emblem Hero and Engage mechanic in the game. As the title says. Any unit equipped/Engaged with Emblem Marth (so far) won't change his/her special skill icon to its appropriate Emblem Hero with ring frame, changes the text to green, and grants additional permanent stats (if merges apply) in the builder Condensed and Echoes stats UI, My Unit however has no issue.

I.e. +1 merged Emblem Marth should grant Caeda +1 HP while Engaged (39 -> 40).

![Screenshot_20240205-064115~2](https://github.com/Eskuero/fehbuilder/assets/37840226/1d23fd39-d158-4ed4-8226-2a6c02732c21)
![Screenshot_20240205-064122~2](https://github.com/Eskuero/fehbuilder/assets/37840226/0c03f9bc-4219-4e2f-a176-a8e30b827f8d)

**Comment by Eskuero** [comment]
Yeah it's not implemented in either yet, they are significantly trickier because the "My Unit" template is built programatically from assets but these ones are sorta harcoded as I modify them manually still.

idk when I will get around to do this because is a pain so no eta
